aws-provider/postgresdb/README.md

```diff
@@ -0,0 +1,382 @@
+# `crossplane` PostgresDB example
+
+In contrast to the other examples in this [AWS provider series](../), this more
+complex example will build a full network hosting an Amazon Postgresql RDS
+database, peered to the cluster VPC with routes between each.
+
+For this example, we'll walk through the process of setting up the IAM roles,
+permission policies and trust relationships to carry out `AssumeRole` operations
+from the service accounts linked to each required Provider-family member within
+the [AWS provider family series](https://marketplace.upbound.io/providers?query=aws).
+
+The approach given in this document utilises [IRSA](https://docs.giantswarm.io/advanced/iam-roles-for-service-accounts/)
+and `AssumeRoleWithWebIdentity` permissions to target a role which grants only
+the permissions required to build the infrastructure required by this example.
+
+By using a targetted role in this manner, you are able to restrict who is
+allowed to use the service account with RBAC permissions although that falls
+outside of the scope of this example.
+
+> **Warning** Creating IAM permissions
+>
+> The  following document assumes that you have permissions to create IAM roles
+> and policies inside your account.
+>
+> If you do not have such permissions, please reach out to your account
+> administrator and ask them to create these for you.
+
+## Create the primary role
+
+For our build to take place, we require the creation of two roles in AWS IAM.
+
+The first of these is an `AssumeRoleWithWebIdentity` which has permissions to
+assume other roles, either in the current account, or using cross-account
+permissions.
+
+For this example, we're going to assume both roles are encompassed in the same
+account.
+`
+1. Edit the file [`crossplane-assume-role-policy`](./policies/crossplane-assume-role-policy.json)
```

Comment:
Maybe let's add a comment, that this policy lists all IAM roles you're allowing to claim?